.eslintignore

```diff
@@ -11,3 +11,4 @@ documentation-site/posts.js
 documentation-site/cheat-sheet.js
 public
 src/codemods/__testfixtures__
+out
```

Comment:
intentional?

```suggestion
```

Reply:
yeah, it was intentional, as eslint fails on the out folder of the extension package